speakers.js

```diff
@@ -0,0 +1,114 @@
+const speakers = [
+  {
+    name: 'Yochai Benkler',
+    honours: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    books: 'Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006',
+    image: {
+      src: 'images/speaker_011.png',
+    },
+  },
+  {
+    name: 'Kilnam Chon',
+    honours: "BBC's 100 businessmen 0f 2021",
+    books: 'Kilnam Chon helped bring the internet to Asia and is an outspoken advocate for the open web and digital com-mons. In 2012. he was inducted into the inaugural class of the Internet Society’s (ISOC) Internet Hall of Fame',
+    image: {
+      src: 'images/speaker_012.png',
+    },
+  },
+];
+
+const seeMoreSpeakers = [
```

Comment:
One array serves the partial display of dynamically generated speaker features. The other serves to display the full speaker list. This array is written like this on purpose.

Reply:
@mailsg I actually understand your point of view, but do you know you can achieve your aim as well with the `fullSpeakerList`  array that has all the needed details? Doing this will ensure that your code follows DRY principle. ℹ️